### ISSUE TITLE
Set Text Color based on Menu Color Theme

### ISSUE THREAD

**Issue by RobLoach** [libretro/RetroArch#6380]
Coming from https://github.com/libretro/RetroArch/pull/6379 , it would be great to set the text color to be black or white automagically when the user flips through the Menu Color themes.

When the user switches to "Light" or "Morning Blue" themes, have the text color switch to black. On all other themes, flip it to white text.
![37260696-8f720258-256c-11e8-94c6-345edc809d70](https://user-images.githubusercontent.com/25086/37266613-c92e344c-2591-11e8-861b-865d804fb559.png)

**Comment by hyarsan** [comment]
Should the same also happen with the icon theme if it's set to Monochrome?

**Comment by RobLoach** [comment]
That's a great idea... Flip between Monochrome and Monochrome Inverted if they're set to either Monochrome?

**Comment by hyarsan** [comment]
Yeah, if the theme is set to Monochrome it will be inverted just like the text.

**Comment by fr500** [comment]
I think themes should be defined by a file instead of the current hardcoded approach.
We would list all the .theme files in the theme selector, and the themes would basically have:

- Font name
- Font colors
- Icon filenames
- Wallpaper filenames

The hardcoded approach we have now is flat out bad, if the assets are missing the themes are still listed.
It would also allow people to add themes without adding more stuff to the codebase.

**Comment by RobLoach** [comment]
The introduction of .theme files sounds like a great idea. Live in [RetroArch Assets](https://github.com/libretro/retroarch-assets)?

**Comment by blizzrdof77** [comment]
I accidentally changed my theme to white, and now I can't get it back because the text and icons are also white. I've had to restore my system completely. Thus, I say this should be prioritized!
